src/compressed_tensors/compressors/utils/semi_structured_conversions.py

```diff
@@ -0,0 +1,341 @@
+#
+# Modified by Roberto Lopez Castro (roberto.lopez.castro@udc.es).
+# Pulled from nm-vllm/vllm/model_executor/layers/quantization/utils/format_24.py
+#
+# flake8: noqa
+# isort: skip_file
+
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@bfineran can we have a license attached to this code if it's largely copied over from an open-source project (vllm), even though "our" Alex is the author?

Reply:
Good call, I think this got added in automatically when I ran make style, is there a way to turn that off for a certain file??

Reply:
you can skip it with `neuralmagic: no copyright. it's an apache license though so I think usage should be pretty permissable